README.md

```diff
@@ -22,6 +22,64 @@ By group 36 of the course "Software Praktikum"@UZH in FS 22.
 
 Start the application and visit /swagger-ui.html to see a swagger ui with the api documentation.
 
+## Introduction
+World famous card game played with 36 Swiss "Jasskarten". 
+
+## Technologies
+Briefly overview about used main technologies: 
+
+![spring](./doc/img/spring.png)
+![java](./doc/img/gradle.png)
+![rapid api](./doc/img/rapis_api.png)
+
+![mockito](./doc/img/mockito.png)
+![hibarnate](./doc/img/hibernate.png)
+![plant](./doc/img/plant_uml.png)
+
+
+### Spring Framework
+The Rapid Api Development Architecture provided by spring-data-rest was used to develop the api.
+The PagingAndSortingRepositories provide request handling, deserialization, crud on the database,
+serialization and rendering of errors. Validation was implemented with Bean validations and if
+necessary with Spring event handlers. Side effects were also done with Spring event handlers.
+
+### Hibernate, JPA
+
+### Jitsi API
+(move to FE README)
+
+### Mockito
+
+### Git
+
+
+## High-level components
+### UI
```

Comment:
We have no ui in the backend

Reply:
I've done it already. Hope this is okey for you @salwil ?